games-action/polymc/polymc-1.4.0.ebuild

```diff
@@ -90,8 +94,11 @@ src_configure(){
 		-DCMAKE_INSTALL_PREFIX="/usr"
 		# Resulting binary is named polymc
 		-DLauncher_APP_BINARY_NAME="${PN}"
+		# Force Qt5 to avoid accidentaly building the Qt6 version and breaking things
+		-DLauncher_QT_VERSION_MAJOR=5
```

Comment:
Maybe we can introduce `QT_SLOT=5` or something similar to use here and in `QT_DEPS`.